build/media-suite_compile.sh

```diff
@@ -402,24 +402,25 @@ if [[ $mediainfo = y || $bmx = y || $curl != n ]]; then
     _deps=("$MINGW_PREFIX/lib/libunistring.a")
     _check=(libidn2.{{,l}a,pc} idn2.h)
     [[ $standalone == y ]] && _check+=(bin-global/idn2.exe)
-    if do_pkgConfig "libidn2 = 2.3.0" &&
-        do_wget -h e1cb1db3d2e249a6a3eb6f0946777c2e892d5c5dc7bd91c74394fc3a01cab8b5 \
-        "https://ftp.gnu.org/gnu/libidn/libidn2-2.3.0.tar.gz"; then
+    if do_pkgConfig "libidn2 = 2.3.7" &&
+        do_wget -h 4c21a791b610b9519b9d0e12b8097bf2f359b12f8dd92647611a929e6bfd7d64 \
+        "https://ftp.gnu.org/gnu/libidn/libidn2-2.3.7.tar.gz"; then
         do_uninstall "${_check[@]}"
+        do_pacman_install gtk-doc
         [[ $standalone == y ]] || sed -ri 's|(bin_PROGRAMS = ).*|\1|g' src/Makefile.am
         # unistring also depends on iconv
         grep_or_sed '@LTLIBUNISTRING@ @LTLIBICONV@' libidn2.pc.in \
             's|(@LTLIBICONV@) (@LTLIBUNISTRING@)|\2 \1|'
-	do_autoreconf   
+	    AUTOPOINT=true do_autoreconf
```

Comment:
why does the indentation here look weird?

Reply:
Looks fine on my end
![{034C0CDA-066C-4585-B6A5-9951FFFA30E9}](https://github.com/user-attachments/assets/6d4e9fbe-379d-4840-a7ed-b1a5f194b118)

Reply:
lol.